rviz_default_plugins/src/rviz_default_plugins/displays/tf/tf_display.cpp

```diff
@@ -287,9 +298,33 @@ void TFDisplay::updateFrames()
 {
   typedef std::vector<std::string> V_string;
   V_string frames = context_->getFrameManager()->getAllFrameNames();
-  std::sort(frames.begin(), frames.end());
 
-  S_FrameInfo current_frames = createOrUpdateFrames(frames);
+  // filter frames according to white-list and black-list regular expressions
+  auto it = frames.begin(), end = frames.end();
```

Comment:
I can't say I'm a huge fan of this syntax.  Maybe:

```suggestion
  V_string::iterator it = frames.begin();
  V_string::iterator end = frames.end();
```